utilities/blob_db/blob_db_impl.cc

```diff
@@ -1459,9 +1459,9 @@ Status BlobDBImpl::GCFileAndUpdateLSM(const std::shared_ptr<BlobFile>& bfptr,
     return s;
   }
 
-  auto* cfh =
+  auto cfh =
       db_impl_->GetColumnFamilyHandleUnlocked(bfptr->column_family_id());
-  auto* cfd = reinterpret_cast<ColumnFamilyHandleImpl*>(cfh)->cfd();
+  auto* cfd = reinterpret_cast<ColumnFamilyHandleImpl*>(cfh.get())->cfd();
```

Comment:
Change this to call `DefaultColumnFamily()` instead since BlobDB don't support CFs for now.